talk/basicconcepts/functions.tex

```diff
@@ -265,7 +265,7 @@
       \begin{cppcode*}{gobble=2}
         /// Count number of dilepton events in data.
         /// \param d Dataset to search.
-        unsigned int countDileptons(Data d) {
+        unsigned int countDileptons(Data &d) {
```

Comment:
I would prefer a const reference :) We should split the const chapter and move some parts of it to basics and OOP ..